src/main/java/frc/robot/commands/ManualPrepScoreSequence.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.Lift.LiftPosition;
+import frc.robot.utils.ScoringLocationUtil;
+import frc.robot.utils.ScoringLocationUtil.ScoreCol;
+import frc.robot.utils.ScoringLocationUtil.ScoreHeight;
+
+/** takes the column and height from ScoringLocationUtil.java
+ * and converts that to a LiftPosition
+ * then gives the position to the given lift */
+public class ManualPrepScoreSequence extends SequentialCommandGroup {
```

Comment:
Maybe move this logic into Lift itself? Give Lift access to `ScoringLocationUtil`